src/js/index.js

```diff
@@ -54,7 +55,8 @@ function handleTags() {
 	}
 
 	Array.from(contentEl.querySelectorAll('.item')).forEach(el => {
-		const show = tags.reduce((sho, tag) => sho || el.classList.contains(`tag-${tag}`), false);
+		const show = (showExpired || !el.classList.contains('tag-expired')) &&
```

Comment:
you have to use `showExpired.checked`

Reply:
I thought I've already addressed this, but for some reason this hasn't been fixed yet. Thanks for reminding me about this.

Reply:
Fixed in 9a9e23925c9a109afa6e292b7c556cb5a13c44e2.